python/paddle/fluid/framework.py

```diff
@@ -2756,6 +2758,9 @@ def __init__(self):
         # the distributed lookup table names
         self._distributed_lookup_table = None
 
+        # the original program before collective transpiling
+        self.origin_program = None
```

Comment:
Do we really need to store the `origin_program` in such a way? It seems too fancy and hard-code.

Reply:
we really need suggestion! we are explore a new distributed algorithm named LocalSGD, which do local update with original non-transpiled program for multiple steps, and then update with the transpiled program.

Reply:
I found that it is better to store the original program in our training code but not in the framework!
we will fix it. Thanks!

Reply:
done